packages/flutter/test/widgets/selectable_region_test.dart

```diff
@@ -73,6 +73,37 @@ void main() {
       await gesture.up();
     }, skip: kIsWeb); // https://github.com/flutter/flutter/issues/102410.
 
+    testWidgets('Do not crash when work with Navigator pages', (WidgetTester tester) async {
```

Comment:
Nit:

'Do not crash when work with Navigator pages'

=>

"Does not crash when using Navigator pages"